src/Microsoft.AspNet.Mvc.Localization/IHtmlLocalizerOfT.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc.Localization
+{
+    /// <summary>
+    /// An <see cref="IHtmlLocalizer"/> that provides strings for <see cref="TResourceSource"/>.
+    /// </summary>
+    /// <typeparam name="TResourceSource">The <see cref="System.Type"/> to provide strings for.</typeparam>
```

Comment:
Since the type parameter is used only to change resource keys, I suggest `<typeparam name="TResource">The <see cref="System.Type"/> to scope the resource names.</typeparam>`. "prefix" might be better than "scope"; up to you.